server/controllers/requestController.js

```diff
@@ -38,4 +38,20 @@ export default class Requests {
         status: 'success',
       });
   }
+  /**
+   * gets all request
+   * @param {Object} req
+   * @param {Object} res
+   * @returns {Object} gets all request
+   * @memberof Request
+   */
+  static getAllRequests(req, res) {
+    return res.status(200)
+      .json({
+        data: {
+          request: requestsDb,
+        },
+        status: 'success',
```

Comment:
Where is your `message`?

Reply:
oh, didnt think it was compulsory, i will add it